terraform/README.md

```diff
@@ -0,0 +1,38 @@
+# Using Terraform
+
+
+
+### Sub-directories and state files
+Each subdirectory under the `terraform/` directory contains an isolated set of terraform configuration files, with the expectation of a few symlinked files, and has a separate state file associated with it.  State files are stored with in S3 under the bucket titled `tfstate`.
```

Comment:
Nit, s/with in/in/ ?
re state files, looks like the bucket is callled tf-base, with data in the tf_state prefix.